tests/cypress/latest/fixtures/custom_cloud-config_upgrade.yaml

```diff
@@ -3,9 +3,34 @@ config:
     users:
     - name: root
       passwd: r0s@pwd1
+    write_files:
+      - path: /etc/ssh/sshd_config
```

Comment:
If I'm correct the `Stable` version of Elemental OS image are now based on SLMicro6.0 no? In that case the path for sshd config should be `/etc/ssh/sshd_config.d/root_access.conf`.